python_modules/dagster-graphql/dagster_graphql/schema/instigation.py

```diff
@@ -267,11 +299,14 @@ def resolve_evaluationResult(self, graphene_info: ResolveInfo):
 
 
 class GrapheneTickEvaluation(graphene.ObjectType):
+    dynamicPartitionRequests = graphene.List(graphene.NonNull(GrapheneDynamicPartitionRequest))
```

Comment:
nit: dynamicPartitionRequests -> dynamicPartitionsRequests to mirror `execution_data.dynamic_partitions_requests`